generators/resources/scaffolderMapping.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
Im not sure what the overall design is or if there is more to come, but Im _very_ against this file. The whole point of a service registry was/is to remove all of the hard-coding of these strings and make everything dynamic/ data driven

Reply:
@gconan Valid point but this file is temporary and I will explain more in detail. Basically, the java is still transitioning their code so making a hard fast break to use keys genenrated from scaffolder will not work. Originally, I was going to use the keys from scaffolder to be the driving force for the folder names, service names, etc, but this can introduce some breaking change.